packages/blade/src/components/ActionList/docs/propsTable.tsx

```diff
@@ -0,0 +1,62 @@
+import type { ActionListItemProps, ActionListSectionProps } from '../ActionListItem';
+import type { ActionListProps } from '../ActionList';
+import { Code } from '~components/Typography';
+import { ScrollLink } from '~src/_helpers/storybook/ScrollLink';
+
+type PropsTableType<T> = Record<
+  keyof Omit<T, 'testID' | '_hideDivider' | '_index'>,
+  string | JSX.Element | { note: string; type: string | JSX.Element }
+>;
+const actionListPropsTables: {
```

Comment:
true but we anyway have to maintain props and their types separately in storybook file to support appropriate stories.

That was in fact a bit more difficult than this because we had to create fake props like "actionListItemLeadingIcon" to match it to `<ActionListItem leading={<ActionListItemIcon />} />` component. 

This will be learning curve for anyone working but still simpler to maintain that the storybook controls mapping